tools/StaticAnalysis/Exceptions/Az.Peering/ExtraAssemblies.csv

```diff
@@ -1 +0,0 @@
-Directory,AssemblyName,Severity,ProblemId,Description,Remediation
```

Comment:
remove tools/StaticAnalysis/Exceptions/Az.Peering/ExtraAssemblies.csv and tools/StaticAnalysis/Exceptions/Az.Peering/MissingAssemblies.csv directly

Reply:
All done.